src/main/java/org/mule/extension/db/internal/domain/connection/mysql/MySqlConnectionParameters.java

```diff
@@ -125,4 +134,13 @@ private void addMuleLoggerProperty(Map<String, String> connectionProperties) {
       }
     }
   }
+
+  private Class<?> getAvailableInterface() throws ClassNotFoundException {
+    try {
+      return Class.forName("com.mysql.cj.log.Log");
+    } catch (ClassNotFoundException ex) {
+      return Class.forName("com.mysql.jdbc.log.Log");
```

Comment:
catch CNFE again and throw a meaningul error saying that neither class was found and an unsupported driver was provided